content/en/observability_pipelines/destinations/_index.md

```diff
@@ -34,6 +34,26 @@ Select and set up your destinations when you [set up a pipeline][2]. This is ste
     {{< nextlink href="observability_pipelines/destinations/#sumo-logic-hosted-collector" >}}Sumo Logic Hosted Collector{{< /nextlink >}}
 {{< /whatsnext >}}
 
+### Event batching {#event-batching-intro}
+
+Observability Pipelines destinations send events in batches to the downstream integration. A batch of events is flushed when one of the following parameters is met:
+
+- Maximum number of events
+- Maximum number of bytes
+- Timeout (seconds)
+
+For example, if a destination's parameters are:
+
+- Maximum number of events = 2
+- Maximum number of bytes = 10,0000
```

Comment:
```suggestion
- Maximum number of bytes = 100,000
```

Reply:
Not sure if it's better to move the comma or delete a zero here. Feel free to update as needed.

Reply:
Thanks for catching that! Moving the comma is correct. Committed you suggestion.